lib/cloudshaper/command.rb

```diff
@@ -8,32 +8,29 @@ class Command
     def initialize(stack, command)
       @stack = stack
       @command = options_for(command)
-      prepare
     end
 
+    # fixme - make these shell safe
     def env
       vars = {}
-      @stack.variables { |k, v| vars["TF_VAR_#{k}"] = v[:default] }
+      @stack.variables.each { |k, v| vars["TF_VAR_#{k}"] = v }
```

Comment:
Actually, I think this is unnecessary because string interpolation should automatically convert symbols to strings?

Either way, I should actually write some specs to make sure these command strings are what we expect, and can accept either strings or symbols.